vendor/bat-native-ledger/src/bat/ledger/internal/endpoint/uphold/get_capabilities/get_capabilities.cc

```diff
@@ -82,13 +85,16 @@ std::map<std::string, bool> GetCapabilities::ParseBody(
     for (const auto& item : list_value->GetList()) {
       const auto* key = item.FindStringKey("key");
       const auto enabled = item.FindBoolKey("enabled");
+      const auto* requirements = item.FindListKey("requirements");
 
-      if (!key || !enabled) {
+      if (!key || !enabled || !requirements) {
```

Comment:
If the requirements field is not present (or not a list), should we treat that the same as an empty requirements field or a non-empty requirements field?

Reply:
The `requirements` array should always be present — I think we should not rely on any implicit assumptions if it's not, since then we wouldn't be able to make sure it's empty.